ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/management-api-gnmi.j2

```diff
@@ -1,18 +1,30 @@
 {# Management API GNMI #}
-{% if management_api_gnmi is defined and management_api_gnmi is not none %}
+{% if management_api_gnmi is arista.avd.defined and management_api_gnmi is not none %}
 
 ## Management API GNMI
 
 ### Management API GNMI Summary
 
-{%     if management_api_gnmi.enable_vrfs is defined %}
-| VRF with GNMI | OCTA |
-| ---------- | ---------- |
+| VRF | OCTA | Notification |
+| --- | ---- | ------------ |
+{%     if management_api_gnmi.octa is arista.avd.defined(true) %}
+{%         set octa = 'enabled' %}
+{%     else %}
+{%         set octa = 'disabled' %}
+{%     endif %}
+{%     if management_api_gnmi.notification_timestamp.send_time is arista.avd.defined(true) %}
+{%         set send_time = 'send-time' %}
+{%     else %}
+{%         set send_time = 'last-change-time' %}
+{%     endif %}
+{%     if management_api_gnmi.enable_vrfs is not arista.avd.defined %}
```

Comment:
I understand that you were going with the default behavior of eos_cli. But in this AVD role, we print only the input given by user. So if enable_vrfs is defined then only we print the values.

Reply:
I don't think I understand this, the `default` vrf is a vrf too, so imo we should document if you have GNMI enabled in default vrf, plus the other things like what's the send-time or if OCTA is enabled, I don't think we should only render a table if someone uses a non-default vrf, what's the use case?